### ISSUE TITLE
why the loss increase much after add nnieqat?

### ISSUE THREAD

**Issue by zimurui** [aovoc/nnieqat-pytorch#24]
thank you for yourshare.....  I add nnieqat on my segmentation training task which has already convergence, but the loss become bigger after some batch, and the segmentation not work...,  which may cause this problem?
